src/native/public/mono/metadata/details/image-functions.h

```diff
@@ -42,8 +42,8 @@ MONO_API_FUNCTION(mono_bool, mono_image_is_dynamic, (MonoImage *image))
 MONO_API_FUNCTION(char*, mono_image_rva_map, (MonoImage *image, uint32_t rva))
 
 MONO_API_FUNCTION(const MonoTableInfo *, mono_image_get_table_info, (MonoImage *image, int table_id))
-MONO_API_FUNCTION(int, mono_image_get_table_rows, (MonoImage *image, int table_id))
-MONO_API_FUNCTION(int, mono_table_info_get_rows, (const MonoTableInfo *table))
+MONO_API_FUNCTION(uint32_t, mono_image_get_table_rows, (MonoImage *image, int table_id))
+MONO_API_FUNCTION(uint32_t, mono_table_info_get_rows, (const MonoTableInfo *table))
```

Comment:
Not these, either

Reply:
> Why is the 3rd step necessary?

I thought there might be a warning in the body because `table_info_get_rows` returns an unsigned.  But if there's no warning, there's nothing to change.